packages/apollo-engine-reporting/src/agent.ts

```diff
@@ -203,6 +192,45 @@ export interface EngineReportingOptions<TContext> {
    * TODO(helen): LINK TO EXAMPLE FUNCTION? e.g. a function recursively search for keys to be blocklisted
    */
   sendVariableValues?: VariableValueOptions;
+  /**
+   * This option allows configuring the behavior of request tracing and
+   * reporting to [Apollo Graph Manager](https://engine.apollographql.com/).
+   *
+   * By default, this is set to `true`, which results in *all* requests being
```

Comment:
should the docs be more clear that this is still only true if an API key is configured via option or env var?